README.md

```diff
@@ -15,6 +15,8 @@ This repo provides wallet builders a pre-made class with all required wallet fun
 - Change `icon` to your wallet icon (pay attention to the required format)
 
 - Change `window.aptos` to be `window.<your-wallet-name>`
+
+> **_NOTE:_** make sure the `name` prop is the same as the `window.<name>` as the adapter is looking for your `name` prop when trying to detect a wallet in the browser. i.e if your `AptosWalletName` is `Aptos`, the `name` in `window.<name>` should be `Aptos`.
```

Comment:
Let's make this consistent so `your-wallet-name` is the same as `<name>` below.

Let's also change the example of `AptosWalletName` is `Aptos` to `AptosWalletName` is `Petra`.  This removes any confusion where English isn't a first language, and you're like, of course my wallet is an `Aptos` wallet.  Versus, the wallet name being the actual wallet name e.g. Petra, Martian, Rise etc.

Reply:
Let's reward it as...

```
**_NOTE:_** Ensure the `name` prop is the same as the `window.<name>`.  The adapter will look for the matching name when detecting a wallet.  For example, if your wallet's name prop is `Petra`, then the window should be `window.petra`.

Additionally, this is a little confusing to me, is the property `AptosWalletName` or `name`?
```

Reply:
Yes it is a bit confusing. The only thing I am trying to say is that the [window object key](https://github.com/aptos-labs/wallet-adapter-plugin-template/blob/main/src/index.ts#L18) should be the same as the `name` [prop](https://github.com/aptos-labs/wallet-adapter-plugin-template/blob/main/src/index.ts#L27). 
i.e if `name = "Aptos"` then the window object key should also be `aptos` (and if `name = "Petra"` then the window object key should also be `petra`.

Now that I am thinking (and writing) about it, we can simply have the `name` prop as a `string` type and remove the us of `AptosWalletName` const (that would be a breaking change)

Reply:
I agree with Greg, let's avoid "Aptos", people will get confused because "Aptos" isn't a wallet.

Also the text says:
> if your `AptosWalletName` is `Aptos`, the `name` in `window.<name>` should be `Aptos`.

But shouldn't it be:
> if your `AptosWalletName` is `Aptos`, the `name` in `window.<name>` should be `aptos`.

And for completeness:
> if your `AptosWalletName` is `Aptos`, the `name` in `window.<name>` should be `aptos`, i.e. `window.aptos`.

Reply:
Yes to all! thanks!